### PR TITLE
Update Readme explaining #48

### PR DESCRIPTION
**Pull request by MatejMecka**
At First people will think there is an error when running it. So i'm sending this Pull Request in order to address the issue and that it's perfectly ok 

This should close: #48

### CODE REVIEW

**Comment by renatomefi** [comment]
But how do we configure it to run our information in dev mode? I think this information is rather valuable!

Thanks

**Comment by MatejMecka** [comment]
@renatomefi Can you check if it's ok?

**Comment by renatomefi** [comment]
@MatejMecka yes, great, thanks!

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ Configuration file: /octocat/personal-website/_config.yml
 
 Don't worry about the GitHub API errors. They are caused because it's run in a dev enviornment and not on GitHub Pages.
 
+You can set a GitHub API Token by setting the JEKYLL_GITHUB_TOKEN to your personal access token.
```
Comment (abhisheksoni27) [inline README.md]: Might wanna add backticks around JEKYLL_GITHUB_TOKEN. (Like this: `JEKYLL_GITHUB_TOKEN `)

Comment (MatejMecka) [reply README.md]: Implemented

Comment (abhisheksoni27) [reply README.md]: 👍

**Comment by MatejMecka** [comment]
@brandonrosage Can you review this PR so it can be merged?

**Comment by MatejMecka** [comment]
Yes it looks perfect!

**Pull request merged**